Assets/HoloToolkit/Sharing/Tests/ImportExportAnchorManager.cs

```diff
@@ -116,18 +116,19 @@ public bool AnchorEstablished
     /// </summary>
     RoomManagerAdapter roomManagerCallbacks;
 
-    void Start()
+    void Awake()
```

Comment:
Changing this to Awake can cause issues accessing a Singleton Instance. There's no guarantee that the Singleton object will be initialized until every object's Awake is called.